tests/api/v1/datadog/api_aws_integration_test.go

```diff
@@ -19,10 +18,9 @@ import (
 	"gopkg.in/h2non/gock.v1"
 )
 
-func generateUniqueAWSAccount(ctx context.Context) datadog.AWSAccount {
-	accountID := fmt.Sprintf("go_%09d", tests.ClockFromContext(ctx).Now().UnixNano()%1000000000)
+func generateUniqueAWSAccount(ctx context.Context, t *testing.T) datadog.AWSAccount {
 	return datadog.AWSAccount{
-		AccountId:                     &accountID,
+		AccountId:                     tests.UniqueEntityName(ctx, t),
```

Comment:
I think accountID needs to be 12 chars exactly, so this probably won't work.
maybe we can have this in the role name, or as a tag in order to identify from where some account ids come from ?

Reply:
Hmm, but the tests are passing, so it seems it does work like this 🤔